src/models/management/commands/seed.py

```diff
@@ -0,0 +1,120 @@
+from django.core.management.base import BaseCommand
```

Comment:
Nothing wrong with this, but I think we do have some sample data, it just doesn't have a nice wrapper around if you were looking for something else!